sigs/sig-serverless.md

```diff
@@ -0,0 +1,152 @@
+## CNCF Serverless SIG Charter
+
+## Introduction
+
+This is the charter referred to in “[CNCF
+SIGs](https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#sig-charter)”
+by the CNCF TOC, and consistent with the [proposed SIG
+definition](https://github.com/cncf/toc/blob/master/sigs/proposed.md).
+
+Borrowing from the [Serverless
+Whitepaper](https://github.com/cncf/wg-serverless/tree/master/whitepapers/serverless-overview),
+Serverless computing refers to the concept of building and running
+applications that do not require server management. Allowing developers
+to focus on writing their applications’ business logic rather than the
+complexities of the platform hosting/managing their applications. As such,
+this SIG will focus on technologies related to enabling this simplified
+developer experience.
+
+## Areas Considered In Scope
+
+Technologies related to hosting, managing, developing and integrating of
+Serverless applications from an application developer's perspective.
+The following are example areas that are considered in scope:
```

Comment:
The scope defined in this charter seems have huge overlap with SIG App Delivery. Actually, if we do `sed s/serverless application/application/g` for this doc, this doc cloud become a perfect charter for SIG App Delivery :-)

I'm not sure how we can handle this in practice when any application layer project comes to review. 

Let's say, somebody what to donate a project named "[Heroku](https://www.heroku.com/)" which defines itself as an application management platform, so naturally, it:
1. [does not require server management for developer](https://www.heroku.com/ctos)
2. [emphasizes on let developers focus on writing their business logic](https://www.heroku.com/developers)
3. has [higher level abstractions](https://devcenter.heroku.com/articles/how-heroku-works#deploy) to abstract complexity away from developers
4. [with a bunch of tooling](https://elements.heroku.com/addons#development) to aid in the development and deployment of application
5. [with great integration technologies](https://elements.heroku.com/addons) that aim to promote interoperability between the various systems and interoperability and connectivity with backend services

Almost 100% align with the below scopes defined in SIG Serverless charter :-) 

So far this charter reads more like a sub-SIG of app delivery, which drops in same scope but emphasizes on serverless app (aka stateless app). This is easy to fix as there are many typical traits for serverless are missing in this charter which we can use for differentiation , for example, scale to zero, event driven, pay as you go etc.

But there's argument that it would narrow down and limit the scope of SIG serverless. In that case, it would be very tricky to fix this confusion unless we can draw a clear boundary between "cloud-native application" and "serverless" application in this charter (can we?).